src/commands/images/removeImageGroup.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from '@microsoft/vscode-azext-utils';
+import * as vscode from 'vscode';
+import { ext } from '../../extensionVariables';
+import { ImageGroupTreeItem } from '../../tree/images/ImageGroupTreeItem';
+import { multiSelectNodes } from '../../utils/multiSelectNodes';
+
+export async function removeImageGroup(context: IActionContext, node?: ImageGroupTreeItem, nodes?: ImageGroupTreeItem[]): Promise<void> {
+    nodes = await multiSelectNodes(
+        { ...context, suppressCreatePick: true, noItemFoundErrorMessage: vscode.l10n.t('No image groups are available to remove') },
+        ext.imagesTree,
+        ImageGroupTreeItem.contextValue,
+        node,
+        nodes
+    );
+
+    const confirmRemove = vscode.l10n.t('Are you sure you want to remove the selected image groups? If there are other tags or child images for images within this group, only the tags will be removed.');
+
+    // no need to check result - cancel will throw a UserCancelledError
+    await context.ui.showWarningMessage (confirmRemove, { modal: true }, { title: 'Remove' });
```

Comment:
```suggestion
    await context.ui.showWarningMessage (confirmRemove, { modal: true }, { title: vscode.l10n.t('Remove') });
```

I see the same mistake was already made in `removeImage.ts`, can you fix it there as well?

Reply:
Absolutely!